### ISSUE TITLE
`clean_tmp_dir` causes conflicts with multiple ingests running at once

### ISSUE THREAD

**Issue by mcampos-quinn** [BAM-PFA/edith#41]
When more than one user runs an ingest at once, the first ingest process to call `utils.clean_tmp_dir()` @ line [399](https://github.com/BAM-PFA/edith/blob/b55df5e093d2bc9eb92b83382cf27242229a9084/edith/app/ingest/ingestProcesses.py#L399) of `ingestProcesses` causes a conflict by deleting JSON files that any other processes would later rely on. 

I commented out that line, but it needs a better solution. This could be solved by treating each ingest process (in pymm and in EDITH!) as an object and using `clean_tmp_dir` as a method only on that object's properties (i.e., the relevant JSON files to that object only get deleted).

So basically I just need to refactor the entire app :)

Temporarily, I can just `os.remove()` the relevant JSON files....

**Comment by mcampos-quinn** [comment]
solved on `ingest-refactor` @ da99feb with a simple call to os.remove() an object's temp json file

**Comment by kieranjol** [comment]
I need to do similar refactors...

**Comment by mcampos-quinn** [comment]
It's been kind of a slog, but pretty manageable once I got the logic reworked. I'm *hoping* it can simplify the logic/file flow but we'll see how it turns out....
